0TA_Template_Sketch/OTA.h

```diff
@@ -9,16 +9,19 @@
 
 #include <WiFiUdp.h>
 #include <ArduinoOTA.h>
-#include <credentials.h>
+#include "credentials.h"
```

Comment:
That is not a good idea. `credentials.h` should be global (shared library in Arduino IDE or PIO). If you make it local chances are that your will ends up to commit your password at one moment or another.